### PR TITLE
Initial version of Angular2/TypeScript sample

### PR DESCRIPTION
**Pull request by vladima**
TypeScript dependency in this PR should be updated after TypeScript 1.5 is released.

### CODE REVIEW

File: angular2/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "typescript-angular2",
+    "version": "1.0.0",
+    "description": "Angular 2/TypeScript demo",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/TypeScriptSamples.git"
```
Comment (mhegazy) [inline angular2/package.json]: I would hold off on adding this sample untill this version is shipped. We do not want confuse ppl about what works where.

Comment (vladima) [inline angular2/package.json]: sure, didn't plan to check this in as is until the release is done. Intent was to suggest the structure and content of the sample

**Comment by vladima** [comment]
pinging @mhegazy

**Comment by mhegazy** [comment]
:+1:

**Comment by vladima** [comment]
Fixes #11

**Pull request merged**